diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter2/comments/CommentsRule.kt

```diff
@@ -70,8 +73,19 @@ class CommentsRule(private val configRules: List<RulesConfig>) : Rule("comments"
                         offset to ktPsiFactory.createImportDirective(ImportPath.fromString(text.substringAfter("$importKeyword "))).node
                     text.contains(packageKeyword) ->
                         offset to ktPsiFactory.createPackageDirective(FqName(text.substringAfter("$packageKeyword "))).node
-                    else ->
-                        offset to ktPsiFactory.createBlockCodeFragment(text, null).node
+                    else -> {
+                        var newNode = ktPsiFactory.createBlockCodeFragment(text, null).node
+                        val isPossibleError = newNode.findChildByType(BLOCK)!!.firstChildNode.elementType == BINARY_EXPRESSION
+                        // A check is performed for the presence of a BINARY_EXPRESSION as first child,
+                        // because the one-line copyright is perceived by the parser as a BINARY EXPRESSION
+                        if (isPossibleError) {
+                            val nodeInsideClass = ktPsiFactory.createBlockCodeFragment("class A {$text}", null).node
```

Comment:
It's still not a universal solution, because it'll help only to avoid invalid top-level declarations. With `fun foo() { $text }` you'll probably still get no `ERROR_ELEMENT`s